lib/faraday/adapter/em_synchrony.rb

```diff
@@ -26,6 +26,11 @@ def call(env)
         # Queue requests for parallel execution.
         if env[:parallel_manager]
           env[:parallel_manager].add(request, http_method, request_config(env)) do |resp|
+            if (req = env[:request]).stream_response?
```

Comment:
I suppose that's ok if we make tests work so we don't break everything later refactoring the code 😁